Gemfile

```diff
@@ -145,6 +145,7 @@ group :development, :test do
   gem "cypress-rails", "~> 0.5.0" # For end to end tests (E2E)
   gem "dotenv-rails", "~> 2.7.6" # For loading ENV variables locally
   gem "faker", "~> 2.17" # A library for generating fake data such as names, addresses, and phone numbers
+  gem "fakeredis", "~> 0.8.0" # Fake (In-memory) driver for redis-rb. Useful for testing environment and machines without Redis.
```

Comment:
👏 Used this at a previous gig, worked very well back then.

Reply:
Yeah, it's working. The only thing that's failing are tests that use `rpush` and `modis` as it uses an internal redis based connection pool and some tricks that `fakeredis` doesn't fully support, but I'm leaning towards mocking `rpush` as we shouldn't test that library anyway :D